projects/redemption_configs/src/configs/generators/python_spec.hpp

```diff
@@ -238,6 +238,12 @@ void write_type(std::ostream& out, type_enumerations&, type_<types::list<T>>, L
     out << "string(default='" << impl::quoted2(s) << "')";
 }
 
+template<class T>
+void write_type(std::ostream& out, type_enumerations&, type_<types::file_permission>, T const & x)
+{
+    out << "file_permission(default='" << impl::stringize_integral(x) << "')";
```

Comment:
`file_permission` type doesn't exist in python parser library (configparser?). Is this function really used?

Reply:
If I don't add this function, I will get compile error :/